server/src/services/email.service.ts

```diff
@@ -94,7 +95,7 @@ export class EmailService {
           ${sender} has invited you to use ToolJet. Use the link below to set up your account and get started.
           </span>
           <br>
-          <a href="<%= @url %>">${`${this.TOOLJET_HOST}/invitations/${invitationtoken}`}</a>
+          <a href="${inviteUrl}">${inviteUrl}</a>
```

Comment:
Can we change `href="${inviteUrl}"` to `href={inviteUrl}`?

Reply:
No. It takes it as a string. because the whole content is enclosed in a back tick.